modular_skyrat/modules/xenos_skyrat_redo/code/xeno_types/defender.dm

```diff
@@ -0,0 +1,109 @@
+/// SKYRAT MODULE SKYRAT_XENO_REDO
+
+/mob/living/carbon/alien/humanoid/skyrat/defender
+	name = "alien defender"
+	caste = "defender"
+	maxHealth = 300
+	health = 300
+	icon_state = "aliendefender"
+	melee_damage_lower = 25
+	melee_damage_upper = 30
+	/// Holds the basic charge ability that the defender will be granted
+	var/datum/action/cooldown/mob_cooldown/charge/basic_charge/defender/charge
+	/// Holds the wrecking ball tail sweep that the defender will be granted
+	var/datum/action/cooldown/spell/aoe/repulse/xeno/crushing/tail_sweep
+	next_evolution = /mob/living/carbon/alien/humanoid/skyrat/warrior
+
+/mob/living/carbon/alien/humanoid/skyrat/defender/Initialize(mapload)
+	. = ..()
+	tail_sweep = new /datum/action/cooldown/spell/aoe/repulse/xeno/crushing()
+	tail_sweep.Grant(src)
+
+	charge = new /datum/action/cooldown/mob_cooldown/charge/basic_charge/defender()
+	charge.Grant(src)
+
+	REMOVE_TRAIT(src, TRAIT_VENTCRAWLER_ALWAYS, INNATE_TRAIT)
+
+	add_movespeed_modifier(/datum/movespeed_modifier/alien_heavy)
+
+/mob/living/carbon/alien/humanoid/skyrat/defender/Destroy()
+	QDEL_NULL(charge)
+	QDEL_NULL(tail_sweep)
+	return ..()
+
+/mob/living/carbon/alien/humanoid/skyrat/defender/create_internal_organs()
+	internal_organs += new /obj/item/organ/internal/alien/plasmavessel/small
+	..()
+
+/datum/action/cooldown/spell/aoe/repulse/xeno/crushing
+	name = "Crushing Tail Sweep"
+	desc = "Throw back attackers with a sweep of your tail, likely breaking some bones in the process."
+
+	cooldown_time = 60 SECONDS
+
+	aoe_radius = 1
+
+	icon_icon = 'modular_skyrat/modules/xenos_skyrat_redo/icons/xeno_actions.dmi'
+	button_icon_state = "crush_tail"
+
+	sparkle_path = /obj/effect/temp_visual/dir_setting/tailsweep/defender
+
+/datum/action/cooldown/spell/aoe/repulse/xeno/crushing/IsAvailable()
+	. = ..()
+	if(!isalien(owner))
+		return FALSE
+	var/mob/living/carbon/alien/humanoid/skyrat/owner_alien = owner
+	if(owner_alien.unable_to_use_abilities)
+		return FALSE
+
+/datum/action/cooldown/spell/aoe/repulse/xeno/crushing/cast_on_thing_in_aoe(atom/movable/victim, atom/caster)
+	if(isalien(victim))
+		return
+	var/turf/throwtarget = get_edge_target_turf(caster, get_dir(caster, get_step_away(victim, caster)))
+	var/dist_from_caster = get_dist(victim, caster)
+
+	if(dist_from_caster <= 0)
+		if(isliving(victim))
+			var/mob/living/victim_living = victim
+			victim_living.Knockdown(10 SECONDS)
+			victim_living.apply_damage(30, BRUTE,BODY_ZONE_CHEST, wound_bonus = 20)
+			shake_camera(victim, 4, 3)
+			playsound(victim, 'sound/effects/clang.ogg', 100, TRUE, 8, 0.9) //the defender's tail is literally just a small wrecking ball, CLANG
+			to_chat(victim, span_userdanger("You're slammed into the floor by [caster]'s tail!"))
```

Comment:
Do an early return here to reduce the indentation.

Also, for the damage values, they should be variables on the spell, I reckon.